### PR TITLE
Document building and installing in the README

### PR DESCRIPTION
**Pull request by Ellpeck**
Still pretty WIP, but much better than nothing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
+### From Visual Studio Marketplace
+vscode-flowr is not available on the Visual Studio Marketplace yet. For now, we recommend installing from the artifact, or building from source.
 
-## Extension Settings
-TO DO
+### From GitHub Releases
+There are no official releases yet. Stay tuned!
 
-# Developing
+### From build artifacts
```
Comment (Ellpeck) [inline README.md:12]: Note that I Don't Use Titlecase for these headers. The two headers above only appear that way because they use proper nouns!

**Comment by Ellpeck** [comment]
This also sort of depends on #22 since it mentions stuff added there

File: README.md [hunk 2]
```diff
@@ -1,18 +1,31 @@
...
-# Developing
+### From GitHub Releases
+There are no official releases yet. Stay tuned!
 
-## Git Hooks
+### From build artifacts
+You can easily download the most recent build of the extension by heading to the [Actions tab](https://github.com/Code-Inspect/vscode-flowr/actions/workflows/package.yml), where you will find a list of runs. Selecting the most recent run will display a summary of it, at the bottom of which you can find the Artifacts section and the `Extension vsix` artifact. Download it und unzip it.
+
+From Visual Studio Code, open the Extensions tab and click on the three dots in the top right to select "Install from VSIX..." Alternatively, you can use the Command Palette to select the option directly. Then, you can select the `vscode-flowr-[version].vsix` file you unzipped to install it.
+
```
Comment (EagleoutIce) [inline README.md:18]: Can you please add a section showing how to use the extension with a simple example? (e.g., open R file, click variable, ...)

Furthermore, maybe we should add a link to vscode-R as a "recommendation".

File: README.md [hunk 3]
```diff
@@ -1,18 +1,31 @@
...
+## Developing
+
+### Building and running from source
+Opening the repository in Visual Studio Code allows using the [existing launch configurations](https://github.com/Code-Inspect/vscode-flowr/blob/main/.vscode/launch.json) which can launch Visual Studio Code with the extension enabled. To use them, open the Run and Debug view and press the Run button at the top, or use the F5 shortcut to start debugging.
```
Comment (EagleoutIce) [inline README.md:22]: npm dependency and `npm ci` missing

Comment (EagleoutIce) [inline README.md:22]:
```suggestion
Opening the repository in Visual Studio Code allows using the [existing launch configurations](https://github.com/Code-Inspect/vscode-flowr/blob/main/.vscode/launch.json) which can launch Visual Studio Code with the extension enabled. To use them, open the Run and Debug view and press the Run button at the top, or use the <kbd>F5</kbd> shortcut to start debugging.
```

**Review by EagleoutIce — APPROVED** [review]

**Pull request merged**